packages/notification-managment-spa/src/environments/environment.ts

```diff
@@ -0,0 +1,17 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
+  production: false,
+  API_URL: '',
```

Comment:
Can assign the values from the environment variables here. Or directly use the environment variable values in the code, without defining it in the environment.

```suggestion
  API_URL: process.env.API_URL,
```

Reply:
Yes this looks great

Reply:
Made the change